spec/factories/new_user.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :user do
+    first_name Faker::Name.first_name
```

Comment:
Surround each value generator with `{ }` to generate a new value each time the factory is called. [Here](http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Dynamic_Attributes) is some additional info.